### ISSUE TITLE
Get-SPOSite Cannot do multiple filters

### ISSUE THREAD

**Issue by FeliciaC-1** [MicrosoftDocs/office-docs-powershell#6806]
I am trying to do multiple filters on Get-SPOSite, e.g. -Filter "{Url -notlike '-my.sharepoint.com' -and Template -eq 'SITEPAGEPUBLISHING#0'}"  But this does not seem to work.  

There is no information on MS docs page about this so I'm not sure if this is possible or not.  Microsoft needs to update this page to inform user if it is possible or not: https://docs.microsoft.com/en-us/powershell/module/sharepoint-online/get-sposite?view=sharepoint-ps#inputs

**Comment by get-itips** [comment]
Hello @FeliciaC-1 how are you?
I tested this to see if we need to impact the documentation.
I was able to successfully filter multiple properties:
![image](https://user-images.githubusercontent.com/36666927/99084974-3c774d00-258d-11eb-9645-7f5597a492cd.png)


However, Template isn't allowed to be used as filter:

![image](https://user-images.githubusercontent.com/36666927/99085037-544ed100-258d-11eb-9393-1d38aa271ea5.png)

So I will update the article, many thanks for your feedback.
